pennylane/devices/default_qubit.py

```diff
@@ -204,6 +204,17 @@ def __init__(
 
     @property
     def stopping_condition(self):
+        """.BooleanFn: Returns the stopping condition for the device. The returned
```

Comment:
Is the output actually of `BooleanFn` type?

Reply:
Yep it is.